### PR TITLE
NGWMN branch

### PR DESCRIPTION
**Pull request by wdwatkins**

### CODE REVIEW

**Comment by wdwatkins** [comment]
Very much a WIP

File: R/importNGWMN_wml2.R [hunk 1]
```diff
@@ -120,23 +123,24 @@ importNGWMN_wml2 <- function(input, asDateTime=FALSE, tz=""){
...
+    #some sites don't have a description
+    siteDesc <- xml_text(xml_find_first(featureMembers, ".//gml:description"))
     
-    siteLocs <- strsplit(xml_text(xml_find_all(returnedDoc, ".//gml:pos")), " ")
-    siteLocs <- data.frame(dec_lat_va=as.numeric(siteLocs[[1]][1]), dec_lon_va=as.numeric(siteLocs[[1]][2]), stringsAsFactors = FALSE)
+    siteLocs <- strsplit(xml_text(xml_find_all(featureMembers, ".//gml:pos")), " ")
+    siteLocs <- data.frame(matrix(unlist(siteLocs), nrow=length(siteLocs), byrow=TRUE), stringsAsFactors = FALSE)
+    names(siteLocs) <- c("dec_lat_va", "dec_lon_va")
+    siteLocs <- mutate(siteLocs, dec_lat_va=as.numeric(dec_lat_va), dec_lon_va=as.numeric(dec_lon_va))
+    #siteLocs <- data.frame(dec_lat_va=as.numeric(siteLocs[[1]][1]), dec_lon_va=as.numeric(siteLocs[[1]][2]), stringsAsFactors = FALSE)
```
Comment (jiwalker-usgs) [inline R/importNGWMN_wml2.R]: remove commented code

File: R/readNGWMNdata.R [hunk 2]
```diff
@@ -146,15 +149,14 @@ readNGWMNlevels <- function(featureID){
...
+  sites <- readNGWMNdata(featureID = featureID, service = "featureOfInterest")
   return(sites)
 }
 
 
-
 retrieveObservation <- function(featureID, asDateTime, attrs){
   #will need to contruct this more piece by piece if other versions, properties are added
-  baseURL <- "http://cida.usgs.gov/ngwmn_cache/sos?request=GetObservation&service=SOS&version=2.0.0&observedProperty=urn:ogc:def:property:OGC:GroundWaterLevel&responseFormat=text/xml&featureOfInterest=VW_GWDP_GEOSERVER."
+  baseURL <- "https://cida-test.er.usgs.gov/ngwmn_cache/sos?request=GetObservation&service=SOS&version=2.0.0&observedProperty=urn:ogc:def:property:OGC:GroundWaterLevel&responseFormat=text/xml&featureOfInterest=VW_GWDP_GEOSERVER."
```
Comment (ldecicco-USGS) [inline R/readNGWMNdata.R]: see `setAccess`
add base

https://cida-test.er.usgs.gov/ngwmn_cache/sos?

**Comment by ldecicco-USGS** [comment]
@jiwalker-usgs does this look right? Shouldn't an upstream pull request wipe out tons of these changes?

**Comment by jiwalker-usgs** [comment]
The target is the NGWMN branch, which doesn't have these commits yet.  So this is to be expected.

**Comment by ldecicco-USGS** [comment]
Could you comment out the test you know will fail, and fix the notes and warnings that Appveyor shows (ie..:

importNGWMN_wml2: no visible binding for global variable 'dec_lat_va'
importNGWMN_wml2: no visible binding for global variable 'dec_lon_va'

Undocumented arguments in documentation object 'readNGWMNdata'
'...'
Documented arguments not in \usage in documentation object 'readNGWMNdata':
'featureID'
Undocumented arguments in documentation object 'retrieveFeatureOfInterest'
  '...' 'asDateTime' 'srsName'

**Pull request merged**